src/typings/index.ts

```diff
@@ -174,7 +174,7 @@ export interface BarChartSettings<T> {
 
     // layout: 'vertical' | 'horizontal';
     keySelector: (value: T) => string;
-    acronymSelector?: (value: T) => string;
+    acronymSelector?: (value: T) => string | undefined;
```

Comment:
In the above statement, acronym is required, so this should work, or could you make the required acronym unrequired?

Reply:
the return type of the function can be either a string or undefined. It is used as 
`acronymSelector: item => item.acronym,`
item.acronym is either string or undefined. So, had to do that.

Reply:
@AdityaKhatri